src/accept.jl

```diff
@@ -0,0 +1,41 @@
+function always_accept(partition::Partition)
+    """ Accepts new partition with probability 1.
+    """
+    return 1
+end
+
+
+function pass_acceptance_fn(partition::Partition, acceptance_fn::Function)

```

Comment:
Do we like this name?

Reply:
I have a feeling you don't 😅 On second thought, I don't really like it either. I'll change to "satisfies"?